### DISCUSSION TITLE
{TXT:EDDI mission accepted destinationstation} does not work

### DISCUSSION THREAD

**Discussion by FireSammy** [EDCD/EDDI#2386] [EDDI VoiceAttack Q&A] [answered]
Hello, like the subject says the {TXT:EDDI mission accepted destinationstation} does not work currently, what might be the reason. It used to work just fine

**Comment by Tkael** [comment, marked as answer]
Hi @FireSammy, not every mission has a destination station recorded to the journal and there are even some cases where no mission data is written to the journal at all. Additional context would be helpful in answering this question. 

Your journal files are saved to [Your 'Saved Games' Folder]/Frontier Developments/Elite Dangerous.
Your EDDI log files are saved to %appdata%/EDDI.

If you can retrieve a journal file and a log for a session where this occurred and point out to me which mission it was then I can investigate further.

**Reply by FireSammy** [reply]
%appdata%/EDDI i have never looked for that and tried to type it in windows search but came up as a browser address. I don't understand, checked the saved games/Frontier developments/elite dangerous and all it gives me is a long list of journal files

**Reply by FireSammy** [reply]
[Journal.220807182401.01.log](https://github.com/EDCD/EDDI/files/9573066/Journal.220807182401.01.log)

**Reply by FireSammy** [reply]
I think i know where the problem is EDDI API not connected

**Reply by FireSammy** [reply]
No didnt work. Connecting EDDI API didnt help

**Reply by Tkael** [reply]
The Frontier API has nothing to do with mission tracking and the journal you provided has no missions in-process at start or accepted during the session. Unfortunately, I still don't have enough information to offer any help / explanation because you have not yet provided a clear example where you have observed this occurring.

**Comment by Tkael** [comment]
Moved to #2393.
